app/scenes/Search/Search.tsx

```diff
@@ -171,6 +177,20 @@ class Search extends React.Component<Props> {
     });
   };
 
+  handleSearchTitleOnly = (ev: React.ChangeEvent<HTMLInputElement>) => {
+    this.searchTitleOnly = ev.target.checked;
+    this.fetchResults();
+    // Disable other filters if this filter is enabled
+    if (ev.target.checked) {
+      this.handleFilterChange({
```

Comment:
It's not great UX to just clear out all the users filters and not include them in the search, we'd need to pass these through and add support on the backend.

Reply:
yes. i'll make changes on the backend to support other filters too.